src/enums/nameRequestTypes.ts

```diff
@@ -0,0 +1,11 @@
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+
+/** Enum to show that NR types are a subset of corp types. */
+/** Enum to map NR types to corp types. */
+/** Enum to sync NR types with corp types. */
+export enum NameRequestTypes {
```

Comment:
Haha, pick just one comment :)

Reply:
Can't we have them all?!

Reply:
A comment can be multi-line for sure. But I think if you hover on NameRequestTypes you will only see 1 of the 3 comments. Merge them if you want, lol.